build/yamls/antrea-eks-node-init.yml

```diff
@@ -0,0 +1,72 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  labels:
+    app: antrea
+    component: antrea-node-init
+  name: antrea-node-init
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-node-init
+  template:
+    metadata:
+      labels:
+        app: antrea
+        component: antrea-node-init
+    spec:
+      hostPID: true
+      hostNetwork: true
+      containers:
+        - name: node-init
+          image: gcr.io/google-containers/startup-script:v1
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+          env:
+          - name: STARTUP_SCRIPT
+            value: |
+              #! /bin/bash
+
+              set -o errexit
+              set -o pipefail
+              set -o nounset
+
+              if [ -f /opt/cni/antrea-node-init-status ]; then
+                  echo "Antrea node init already done. Exiting"
+                  exit
+              fi
+
+              while true; do
+                  cni_conf=$(ls /etc/cni/net.d | head -n1)
+                  if [[ ! -z $cni_conf ]]; then break; fi
+                  echo "Waiting for cni conf file"
+                  sleep 2s
+              done
+              cni_conf="/etc/cni/net.d/$cni_conf"
+
+              while true; do
+                  if grep -sq "antrea" $cni_conf; then break; fi
```

Comment:
The presence of antrea in cni conf doesn't mean it has taken effect (cni conf is read every 5 seconds), should it wait for a few seconds in case of race condition? https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/dockershim/network/cni/cni.go#L50

Reply:
When we detect antrea, which happens via init-container, and then kill the container, kubelet schedules a pod when both cni returns a success

Reply:
I can add another 5 sec sleep for safety. However, I didnt see any issue..

Reply:
I think there is an issue, but it can only happen when Pods happen to be created after this line has run and before kubelet reloads the cni conf.

Reply:
Yes updated it for safety